README.md

```diff
@@ -10,6 +10,17 @@ To build this snap locally you need snapcraft. The project must be built as real
 $ sudo snapcraft
 ```
 
+# Writing code
+
+The usual way to add functionality is to write a shell script hook
+with the `.chroot` extenstion under the `hooks/` directory. These hooks
+are run inside the base image filesystem.
+
+Each hook should have a matching `.test` file that validates that the
+`.chroot` file works as expected. Those `.test` scripts will be run
```

Comment:
it is not very clear from this description that these tests will be run after building with snapcraft,  if I under the travis.yml.

Also I wonder whether "sanity" or "check" would be a better name for them and this process.